atest/testdata/variables/automatic_variables/auto1.robot

```diff
@@ -38,15 +38,25 @@ Test Documentation
 
 Test Tags
     [Tags]    id-${42}    Hello, world!    ${VARIABLE}
-    [Setup]    Check Test Tags    Force 1    Hello, world!    id-42    ${VARIABLE}
-    Check Test Tags    Force 1    Hello, world!    id-42    ${VARIABLE}
-    [Teardown]    Check Test Tags    Force 1    Hello, world!    id-42    ${VARIABLE}
+    [Setup]    Check Test Tags    Force 1    Hello, world!    id-42    include this test    ${VARIABLE}
+    Check Test Tags    Force 1    Hello, world!    id-42    include this test    ${VARIABLE}
+    [Teardown]    Check Test Tags    Force 1    Hello, world!    id-42    include this test    ${VARIABLE}
 
 Modifying ${TEST TAGS} does not affect actual tags test has
     [Documentation]    The variable is changed but not "real" tags
     [Tags]    mytag
     Append To List    ${TEST TAGS}    not really added
-    Check Test Tags    Force 1    mytag    not really added
+    Check Test Tags    Force 1    include this test    mytag    not really added
+
+Include-tags Available As Automatic Variables
+    Should Contain    ${OPTIONS.include}    include this test
+
+Exclude-tags Available As Automatic Variables
+    Should Contain    ${OPTIONS.exclude}    exclude this test
+
+Skip-tags Available As Automatic Variables
+    Should Contain    ${OPTIONS.skip}    Skip_Me
+    Should Contain    ${OPTIONS.skiponfailure}    me_too
```

Comment:
It would be good to test that access like `${OPTIONS}[include]` works as well.

Reply:
Noticed now that these tests aren't actually validated by tests on `atest/robot` side. I'll fix that.